src/components/PackTile.js

```diff
@@ -80,7 +81,7 @@ const PackTile = ({
     >
       <PackImg
         logoUrl={logoUrl}
-        packName={name}
+        packName={title || name}
```

Comment:
i think we probably don't need the `or` here if we know title will be required (and possibly name is going away?) lemme know if that's wrong though
```suggestion
        packName={title}
```